templates/patterns/gallery/gallery.ui_patterns.yml

```diff
@@ -0,0 +1,40 @@
+gallery:
+  label: "Gallery"
+  description: "Gallery of available media."
+  fields:
+    items:
+      type: "array"
+      label: "Items"
+      description: "List of GalleryValueObject objects."
```

Comment:
This needs to be changed to `List of GalleryItemValueObject objects.`

Reply:
:+1: